src/SingleProject/Resizetizer/src/nuget/buildTransitive/Microsoft.Maui.Resizetizer.After.targets

```diff
@@ -720,10 +728,17 @@
 
         <!-- Touch/create our stamp file for outputs -->
         <Touch Files="$(_ResizetizerStampFile)" AlwaysCreate="True" />
```

Comment:
So the idea is we write the list of files out to a file, then read them back in on the next build and use that list in the outputs. That way we can detect if files get deleted.

There is an issue where files are being deleted but the stamp file is left behind. I can't seem to track down when or why this happens. It only happens when building for all the platforms it seems (that would be wrong). 

It needs more investigation, but this might be a good fix until we can come up with something better.